packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-run-sample/main/nodejs/testapp.md

```diff
@@ -0,0 +1,32 @@
+## Steps to test the sample app
+
+1. If not already done, set up your Okta org by completing these steps: [Set up your Okta org (for password factor only use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-password-factor-only-use-cases).
```

Comment:
nit: looks like all steps are with `1.`

Reply:
Thanks, I believe this renders in order with Vue.Press but will confirm.